test.js

```diff
@@ -1,4 +1,5 @@
 const tools = require('./fizzbuzz.js');
```

Comment:
convention is to name the variable containing the exported object the same thing as the file (which is hopefully sensibly named as to what it exports). So you'd do this:

`const fizzbuzz = require('./fizzbuzz.js')`